components/Nav/Nav.test.tsx

```diff
@@ -125,4 +125,22 @@ describe('Nav', () => {
 
     expect(mockPush).toHaveBeenCalledWith('/login');
   });
+
+  it('it should close the modal when the signout button is clicked', async () => {
```

Comment:
nit: I would reword this to "... close the drawer..." as that what it is called everywhere in our app.